Example/Tests/Tests.m

```diff
@@ -50,7 +50,7 @@ - (ADBMediaObject *)createWithProperties:(NSDictionary *)properties andEventType
 
     beforeEach(^{
         mockADBMobile = mockClass([ADBMobile class]);
-        integration = [[SEGAdobeIntegration alloc] initWithSettings:@{} andADBMobile:mockADBMobile andADBMediaHeartbeatFactory:nil andADBMediaHeartbeatConfig:nil andADBMediaObjectFactory:nil];
+        integration = [[SEGAdobeIntegration alloc] initWithSettings:@{} adobe:mockADBMobile andMediaHeartbeatFactory:nil andMediaHeartbeatConfig:nil andMediaObjectFactory:nil];
```

Comment:
Doesn't really look like this is used by most of the tests. It should be scoped better so we're only doing this for the tests where we need it.

Alternatively we can just pass in the mock factories here .